### PR TITLE
[ONNX] Fix onnx custom op export & add initial test case

### PR DESCRIPTION
**Pull request by BowenBao**
- Fix typo in ```torch/onnx/utils.py``` when looking up registered custom ops.
- Add a simple test case 
    1. Register custom op with ```TorchScript``` using ```cpp_extension.load_inline```.
    2. Register custom op with ```torch.onnx.symbolic``` using ```register_custom_op_symbolic```.
    3. Export model with custom op, and verify with Caffe2 backend.

### CODE REVIEW

**Comment by BowenBao** [comment]
hi @houseroad, I'm adding test case for custom op support, that requires JIT C++ extensions. The CI for caffe2_onnx_py2_gcc5_ubuntu16_04_test is throwing errors

> Jun 05 00:05:25 >           raise RuntimeError(message)
Jun 05 00:05:25 E           RuntimeError: Error building extension 'custom_add': [1/2] c++ -MMD -MF main.o.d -DTORCH_EXTENSION_NAME=custom_add -DTORCH_API_INCLUDE_EXTENSION_H -isystem /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include -isystem /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/torch/csrc/api/include -isystem /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/TH -isystem /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/THC -isystem /usr/local/include/python2.7 -D_GLIBCXX_USE_CXX11_ABI=1 -fPIC -std=c++11 -c /tmp/torch_extensions/custom_add/main.cpp -o main.o
Jun 05 00:05:25 E           FAILED: main.o 
Jun 05 00:05:25 E           c++ -MMD -MF main.o.d -DTORCH_EXTENSION_NAME=custom_add -DTORCH_API_INCLUDE_EXTENSION_H -isystem /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include -isystem /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/torch/csrc/api/include -isystem /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/TH -isystem /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/THC -isystem /usr/local/include/python2.7 -D_GLIBCXX_USE_CXX11_ABI=1 -fPIC -std=c++11 -c /tmp/torch_extensions/custom_add/main.cpp -o main.o
Jun 05 00:05:25 E           In file included from /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/torch/csrc/Device.h:3:0,
Jun 05 00:05:25 E                            from /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/torch/csrc/api/include/torch/python.h:8,
Jun 05 00:05:25 E                            from /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/torch/extension.h:6,
Jun 05 00:05:25 E                            from /tmp/torch_extensions/custom_add/main.cpp:1:
Jun 05 00:05:25 E           /var/lib/jenkins/.local/lib/python2.7/site-packages/torch/include/torch/csrc/python_headers.h:9:20: **fatal error: Python.h: No such file or directory**
Jun 05 00:05:25 E           compilation terminated.
Jun 05 00:05:25 E           ninja: build stopped: subcommand failed.

Do you have suggestions on how to fix this?

**Comment by jeffreyksmithjr** [comment]
Requesting review from @dzhulgakov and/or @houseroad .

File: scripts/onnx/test.sh [hunk 1]
```diff
@@ -24,6 +24,17 @@ done
 set -- "${UNKNOWN[@]}" # leave UNKNOWN
 
 pip install pytest scipy hypothesis
+# JIT C++ extensions require ninja.
+pip install ninja --user
+# ninja is installed in /var/lib/jenkins/.local/bin
+export PATH="/var/lib/jenkins/.local/bin:$PATH"
+
+sudo apt-get install libpython-dev
+ls -al /usr/local/include
```
Comment (houseroad) [inline scripts/onnx/test.sh]: Remove these debug command?

**Review by houseroad — COMMENTED** [review]
Btw, I am adding/changing the onnx runtime ci.

File: scripts/onnx/test.sh (hunk 1 above)
Comment (BowenBao) [reply scripts/onnx/test.sh]: removed ... from the debug commands it seems the path `/usr/local/include/python2.7` doesn't exist,  that might be the reason why JIT compiler couldn't find `Python.h` file.

**Review by houseroad — APPROVED** [review]
Looks good. It's fine to skip the custom op test in python 2, since py2 is approaching the end of its life.

**Pull request closed without merging**